### DISCUSSION TITLE
other types of question than yes/no

### DISCUSSION THREAD

**Discussion by emilieg126** [iSoron/uhabits#1018] [Help & FAQ] [answered]
I saw somewhere that we can create different types of questions. Like 'how many numbers of pages did you read today' and you enter a number. How? Where? thanks :)

**Comment by ke352802081770314** [comment, marked as answer]
Firstly, check that you have Loop Habit Tracker version 2.0.x (triple dots, about Loop) - This version has brought a massive change in the UI and functionality. If not, obtain it.
In version 2, when you create a new habit using +, immediately you will be prompted to choose between yes/no and measurable. the second choice is the type you are referring to.

**Reply by emilieg126** [reply]
How do you get the version 2.0? The app says I have 1.8.12, but I don't have the option to do any update

**Reply by ke352802081770314** [reply]
#870 the last answer by @iSoron lists two alternative methods to get it.

**Reply by emilieg126** [reply]
awesome! it works, thank you so much :D
